src/adapters/bidderFactory.js

```diff
@@ -194,6 +194,12 @@ export function newBidder(spec) {
         events.emit(CONSTANTS.EVENTS.BIDDER_DONE, bidderRequest);
 
         registerSyncs(responses, bidderRequest.gdprConsent);
+        if (window['owpbjs'] &&
```

Comment:
I am not sure why this change is required, can you please explain?

Reply:
For the 1st time bids are fetched, usersync gets triggered automatically. 
in subsequent calls to fetchBids we are setting the enableOverride property in prebid.js in OpenWrap. 
Once this is set, we need to call  owpbjs.triggerUserSyncs() explicitly after registerSyncs() is called. Hence this change.

Reply:
Please do not make this change in Prebid library we make this change in OpenWrap. 
We should avoid changes in Prebid.

Reply:
We need to call owpbjs.triggerUserSyncs() only after registerSyncs() is called, since this function sets the sync urls in appropriate variables. 
Hence this change is required in prebid. 
Let me know if there is a workaround for this in Openwrap. I'll investigate on this as well.

Reply:
In OpenWrap code, we should keep track of how many times we are calling Prebid library API to fetch bids, a boolean flag is sufficient for this.
On the first call to Prebid Library, we will not call triggerUserSyncs API as user-sync is triggered by Prebid Library for first call.
On subsequent calls to Prebid Library API for fetch bids, we should call triggerUserSyncs API from OpenWrap.
triggerUserSyncs API should be called from bidsBackHandler in OpenWrap code, approx line #492 in OpenWrap/src_new/adapters/prebid.js

Reply:
In OpenWrap code, we should keep track of how many times we are calling Prebid library API to fetch bids, a boolean flag is sufficient for this.
On the first call to Prebid Library, we will not call triggerUserSyncs API as user-sync is triggered by Prebid Library for first call.
On subsequent calls to Prebid Library API for fetch bids, we should call triggerUserSyncs API from OpenWrap.
triggerUserSyncs API should be called from bidsBackHandler in OpenWrap code, approx line #492 in OpenWrap/src_new/adapters/prebid.js